### PR TITLE
Add logs to Log Analytics for Grafana

### PR DESCRIPTION
**Pull request by albal**

### CODE REVIEW

**Review by rgparkins — CHANGES_REQUESTED** [review]
Hi Al, could we do this for the cpd app as well

Thanks

File: Terraform/grafana-log.tf [hunk 1]
```diff
@@ -0,0 +1,22 @@
+resource "azurerm_monitor_diagnostic_setting" "grafana-diagnostics" {
+  name                       = var.gf_diag_name[terraform.workspace]
+  target_resource_id         = azurerm_linux_web_app.grafana-web-app.id
+  log_analytics_workspace_id = data.azurerm_log_analytics_workspace.log-analytics-ws.id
+
+  metric {
```
Comment (rgparkins) [inline Terraform/grafana-log.tf:6]: Do we need this enabled? Does this mean ALL metrics sent (which is not what we want)

Comment (albal) [reply Terraform/grafana-log.tf:6]: It's metrics though not logs.  Do you not want any metrics at all?

Comment (rgparkins) [reply Terraform/grafana-log.tf:6]: ok let deploy and when done I can review dev. thanks

**Review by rgparkins — APPROVED** [review]
all good lets check in lower environments

**Pull request merged**